### PR TITLE
Upgrade deps and add --censor-stats

### PR DESCRIPTION
**Pull request by Rufflewind**
See also: https://github.com/natefaubion/purescript-psa-utils/pull/4

### CODE REVIEW

File: src/Main.purs [hunk 1]
```diff
@@ -313,7 +317,7 @@ Available options:
 -- Due to `catchException` label annoyingness
 catchException'
   :: forall a eff
-   . (Error -> Eff (err :: EXCEPTION | eff) a)
-  -> Eff (err :: EXCEPTION | eff) a
-  -> Eff (err :: EXCEPTION | eff) a
+   . (Error -> Eff (exception :: EXCEPTION | eff) a)
+  -> Eff (exception :: EXCEPTION | eff) a
+  -> Eff (exception :: EXCEPTION | eff) a
```
Comment (natefaubion) [inline src/Main.purs]: This function can be removed altogether since we no longer have duplicate label issues.

Comment (Rufflewind) [inline src/Main.purs]: Removed.

**Comment by mizunashi-mana** [comment]
I ❤ this option.  Any updates？

**Comment by natefaubion** [comment]
Apologies. I will try to look at this again this week.

**Pull request merged**